src/objects.js

```diff
@@ -1,41 +1,52 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    name,
+    age
+  };
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
```

Comment:
The linter has discouraged against the use of the method `.hasOwnProperty`. To tackle this problem without this method you can use the `in` operator. i.e.
```
return property in object
```